Makefile

```diff
@@ -24,6 +24,7 @@ PIP_WRAPPER = $(PY_BIN) -m pip
 export PY38 = $(shell jq -r '.python_versions.PY38' .ci/variables.json)
 export PY39 = $(shell jq -r '.python_versions.PY39' .ci/variables.json)
 export PY310 = $(shell jq -r '.python_versions.PY310' .ci/variables.json)
+export PY311 = $(shell jq -r '.python_versions.PY311' .ci/variables.json)
```

Comment:
Let's switch to `:=` here, see https://github.com/elastic/rally/pull/1686

Reply:
Thanks, done in [`a49ff09` (#1685)](https://github.com/elastic/rally/pull/1685/commits/a49ff09502e3a8375eb4ba61835cd45e64e93839)